src/shared/services/db_user_service.py

```diff
@@ -11,13 +11,13 @@
 from src.shared.db_models.account_history import AccountHistory
 
 
-def create_user(username, email, password):
+def create_user(username, email, password, confirmed = False):
     '''
     Creates user and save into database. Returns True if user is created, else False.
     '''
     user = User(username = username,
             email = email,
-            confirmed = False)
+            confirmed = confirmed)
     try:
         user.set_password(password)
         user.created = datetime.now()
```

Comment:
Correct, it's so that [this command](https://github.com/saasforge/open-source-saas-boilerpate/pull/12/files#diff-9593faa52e480476d0f5eef99f197fbbR29) run before the test suite will succeed in clearing the DB and adding a single test user.